text/3-entrees/testing/mock.md

```diff
@@ -69,9 +71,12 @@ Now, declare the mock runtime as a unit structure
 pub struct TestRuntime;
 ```
 
-The `derive` macro attribute provides implementations of the `Clone + PartialEq + Eq + Debug` traits for the `TestRuntime` struct.
+The `derive` macro attribute provides implementations of the `Clone + PartialEq + Eq + Debug` traits
```

Comment:
```suggestion
The `derive` macro attribute provides implementations of the `Clone`, `PartialEq`, `Eq`, `Debug` traits
```
Given that `Beef + Sheep` is Rust syntax for specifying trait bounds it might be better to just list the traits to avoid confusion?